tests/xcodeproj/Single-Application-Project.xcodeproj/project.pbxproj

```diff
@@ -17,6 +17,7 @@
 		48A8F4B41EF706F8BEEC7ACB /* Single-Application-UnitTests.xctest */ = {isa = PBXFileReference; includeInIndex = 0; lastKnownFileType = wrapper.cfbundle; path = "Single-Application-UnitTests.xctest"; sourceTree = BUILT_PRODUCTS_DIR; };
 		5FF81A9585DBB110EBD9CAC0 /* test.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; name = test.swift; path = test.swift; sourceTree = "<group>"; };
 		8DBB99CE021DEB0B97F196EA /* common.pch */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; name = common.pch; path = ../../rules/library/common.pch; sourceTree = "<group>"; };
+		BDFE1D6BB539968405F979CA /* Single_Application_UnitTests-Swift.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; name = "Single_Application_UnitTests-Swift.h"; path = "Single_Application_UnitTests-Swift.h"; sourceTree = "<group>"; };
```

Comment:
cc @amberdixon on this getting added

Reply:
is this expected?

Reply:
I still don't know why this is getting added, but I'm seeing this in my PRs too. The underlying reason is that the Swift.h file is being treated as a source file (i.e. running `File.is_source` on it returns true), even though the file is actually being generated by one of the rules in rules_swift. Perhaps we can leave it here and make a note to remove it later, after figuring out why it's being added.

Reply:
Do we have expectation that no file change should happen to project file?

Reply:
yes, because there is a check in CI for `git diff --nocheck` ... so if the xcodeproj rule generates a project that is different from the one checked in already, then the CI job will fail.

I'm not sure why the -Swift.h file is showing up in the bazel target's sources as an actual source (because it's really a generated file.) Because it appears to be an actual source, the xcodeproj rule adds this file to the generated xcodeproject.

Reply:
Will deal with this in future: https://github.com/bazel-ios/rules_ios/issues/49